resources/lib/sutils.py

```diff
@@ -131,23 +131,27 @@ def scanRunning(self):
         return (xbmc.getCondVisibility('Library.IsScanningVideo') or
                 xbmc.getCondVisibility('Library.IsScanningMusic'))
 
-    def getBBDB(self, name):
-        name = util.request('http://csfd.bbaron.sk/find.php?' +
-                            urllib.urlencode({'sosac': 1, 'name': name}))
-        if name != '':
-            return self.getTVDB(name, 1)
-        return None
-
-    def getTVDB(self, name, level=0):
-        data = util.request('http://thetvdb.com/api/GetSeries.php?' +
-                            urllib.urlencode({'seriesname': name, 'language': 'cs'}))
-        try:
-            tvid = re.search('<id>(\d+)</id>', data).group(1)
-        except:
-            if level == 0:
-                tvid = self.getBBDB(name)
-            else:
+    def getTVDB(self, name, id):
+        if id:
+            data = (util.request('http://thetvdb.com/api/GetSeriesByRemoteID.php?imdbid=tt' +
+                    id + '&language=all'))
+            try:
+                tvid = re.search('<id>(\d+)</id>', data).group(1)
+            except:
                 tvid = None
```

Comment:
Tu urcite nechces odchytavat kazdu moznu exception. Navise, preco sa tomu nevyhnes napr. takto ?

```
tvid = re.search(...)
if tvid:
    return tvid.group(1)
```

Potom by si nemusel mat o riadok nizssie `if tvid is None:`, ktory vies mimochodom skratit na `if not tvid:`.